transcode.sh

```diff
@@ -48,22 +48,56 @@ die() {
 	exit ${2:-1}
 }
 
-test_dependencies() {
+test_homebrew() {
+  if [ `which brew | wc -l` -lt 1 ]; then
+    die "Homebrew is not installed. Please install Handbrake manually."
```

Comment:
Offer to install Handbrake if the user installs Homebrew.
Or offer to install Homebrew.